roles/kubevirt/tasks/deprovision.yml

```diff
@@ -25,6 +25,7 @@
   get_url:
     url: "{{ release_manifest_url }}/v{{ version }}/kubevirt.yaml"
     dest: "/tmp/kubevirt.yml"
+    force: yes
```

Comment:
This change can cause provision and deprovision with different manifests.

Reply:
Thanks for the review, it's possible cause such a problem. At first I think it would be okay if provision and deprovision with same version.

Reply:
So in order to do that we could add suffix to manifest file ...
```
     dest: "/tmp/kubevirt-v{{ version }}.yml"
```

And change it down further ...

Reply:
Change it like `/tmp/kubevirt-v{{ version }}.yml`, there are more places need to change accordingly. I prefer to use force for `get_url`, it's not harmful.